cli/tests/test_util_command.rs

```diff
@@ -41,18 +41,7 @@ fn test_util_config_schema() {
 #[test]
 fn test_gc_args() {
     let test_env = TestEnvironment::default();
-    // Use the local backend because GitBackend::gc() depends on the git CLI.
-    test_env
-        .run_jj_in(
-            ".",
-            [
-                "debug",
-                "toy-backend-init",
-                "repo",
-                "--config=ui.allow-init-native=true",
-            ],
-        )
-        .success();
+    test_env.run_jj_in(".", ["git", "init", "repo"]).success();
```

Comment:
nit: would be better to move this commit first and avoid the churn here

Reply:
Done.